geomstats/geometry/lie_group.py

```diff
@@ -235,3 +235,32 @@ def group_exponential_barycenter(
 
     def add_metric(self, metric):
         self.metrics.append(metric)
+
+    def lie_bracket(self, base_point, first_tan_vec, second_tan_vec):
```

Comment:
let's put the `base_point` as the last parameter, as we have adopted this implicit convention everywhere else in geomstats

Reply:
let's also write `tangent_vector` to be consistent with the naming elsewhere

(also, it might be obvious for us that tan_vec is tangent_vector but it could not be for future developers, so complete variable names can help!)

Reply:
add `base_point=None` as default value, and make the base point the identity when it's not provided (so that users can write lie_bracket[a, b] without mentioning the base point)

Reply:
for naming consistency, it would actually be `tangent_vector_a` and `tangent_vector_b`.
it would also nicely match your docstring

then: first_term could be a_b_term and second_term could be b_a_term